src/renderer/src/routes/pages/Send/Recipient.tsx

```diff
@@ -13,16 +26,30 @@ export const Recipient = () => {
           <TbStepInto className="text-blue w-5 h-5 mr-3 ml-1" />
           <span>{t('recipientAddress')}</span>
         </div>
-        <div className="flex items-center">
+        <button
+          className="flex items-center"
+          onClick={modalNavigateWrapper('select-contact', {
+            state: {
+              handleSelectContact: handleSelectContact,
+            },
+          })}
+        >
           <TbUsers className="w-5 h-5 text-neon mr-1" />
           <span className="text-neon">{t('contacts')}</span>
-        </div>
+        </button>
```

Comment:
What do you think about creating a new <Button> variant called text for it ? It would be better because we could reuse it elsewhere. Follow the same structure as we have for the other button variants